### PR TITLE
Added support for encoding the URL data

### PR DESCRIPTION
**Pull request by ficocelliguy**
This was a pain, and it doesn't shorten the url by much yet, but it helps.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -143,40 +143,42 @@ app.controller('Damage', function ($scope) {
...
-      reduction: $scope.sections.mitigation.tables.reduction.values.slice(0, -1),
-      flat: $scope.sections.taken.tables.flat.values.slice(0, -1),
+      i: $scope.sections.monster.tables.increase.values.slice(0, -1),
+      m: $scope.sections.monster.tables.more.values.slice(0, -1),
+      s: $scope.sections.shift.tables.shifts.values.slice(0, -1),
+      a: $scope.sections.mitigation.armor,
+      c: $scope.sections.mitigation.charges,
+      r: $scope.sections.mitigation.tables.reduction.values.slice(0, -1),
+      f: $scope.sections.taken.tables.flat.values.slice(0, -1),
       reduced: $scope.sections.taken.tables.reduced.values.slice(0, -1),
```
Comment (PsychicMuffin) [inline app.js]: if we're gonna go this route, might as well do something like d for reduced, and t for resist or w/e

File: index.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8">
   <title>Path of Damage</title>
   <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.4/angular.min.js"></script>
-  <script src="lz-string.min.js"></script>
+  <script src="angular-lz-string.js"></script>
```
Comment (PsychicMuffin) [inline index.html:7]: Do we really need this? Did you test it with just the normal library?

**Review by PsychicMuffin — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (ficocelliguy) [reply index.html:7]: Normal library? You need this here, else Angular can't find it and it breaks.

Comment (PsychicMuffin) [reply index.html:7]: I meant using the non-angular library, but it's aright, I have some tinkering to do later

**Review by PsychicMuffin — APPROVED** [review]

**Pull request merged**